lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -566,6 +566,15 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( empty( $data['capabilities'] ) ) {
+			$data['capabilities'] = new stdClass();
```

Comment:
Rather than using `new stdClass`, can we instead just cast `$user->caps` to an `object` above?